release-notes.md

```diff
@@ -294,6 +300,60 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v35.0
+
+This release has been tracked in our [roadmap
+project](https://github.com/orgs/cloud-hypervisor/projects/6) as iteration
+v35.0. The following user visible changes have been made:
+
+### `virtio-vsock` support for guest kernel v6.3+
```

Comment:
The style for this document is to use "title case" for the headings. I've pushed a fixed version.